client/view/BrowserStats/BrowsersStat.css

```diff
@@ -176,3 +176,7 @@
   font-size: 14px;
   line-height: 20px;
 }
+
+.BrowsersStat__toolLink {
+  color: currentcolor;
```

Comment:
Why not `currentColor` (camelCase) as specification? :)
https://www.w3.org/TR/css-color-3/#currentColor-def

Reply:
This is what our Stylelint config told us (because of Stylelint recommended config).

As I understand because camelCase was a mistake here. I remember some post from CSS author where he regrets from some mistake and camelCase here was one of them.

Reply:
😮